addons/sale_timesheet/models/product.py

```diff
@@ -74,10 +74,13 @@ def _onchange_service_fields(self):
             if record.type == 'service' and record.service_type == 'timesheet' and \
                not (record._origin.service_policy and record.service_policy == record._origin.service_policy):
                 record.uom_id = self.env.ref('uom.product_uom_hour')
+                record.uom_po_id = self.env.ref('uom.product_uom_hour')
             elif record._origin.uom_id:
                 record.uom_id = record._origin.uom_id
+                record.uom_po_id = record._origin.uom_id
             else:
                 record.uom_id = self._get_default_uom_id()
+                record.uom_po_id = self._get_default_uom_id()
```

Comment:
```suggestion
            elif record._origin.uom_id:
                record.uom_id = record._origin.uom_id
            else:
                record.uom_id = self._get_default_uom_id()
            record.uom_po_id = record.uom_id
```

Reply:
@geflx Friday brain are not the best :grimacing: after looking at the code this seems better don't you think ?